Dockerfile

```diff
@@ -39,7 +39,15 @@ ENV BACKUP_SCHEDULE='* * * * *' \
 #-- Other steps
 #--
 RUN apk --no-cache add ca-certificates && update-ca-certificates
-RUN wget https://github.com/gilbertchen/duplicacy/releases/download/v${DUPLICACY_VERSION}/duplicacy_linux_x64_${DUPLICACY_VERSION} -O /usr/bin/duplicacy && \
+RUN ARCH="$(uname -m)";\
+    if [ "$ARCH" == "x86_64" ]; then \
+        DUPLICACY_ARCH="x64"; \
+    elif [ "$ARCH" == "aarch64" ]; then \
+        DUPLICACY_ARCH="arm64"; \
+    elif [ "$ARCH" == "armv7l" ]; then \
+        DUPLICACY_ARCH="arm"; \
+    fi; \
```

Comment:
If $ARCH is none of these then DUPLICACY_ARCH will be undefined - how about in this case we leave it as x64 by default?

Reply:
done!